### PR TITLE
RHMAP-17534 Managing ENV Path in Jenkins Pipelines

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: vars/fhcapNode.groovy [hunk 1]
```diff
@@ -25,58 +25,54 @@ def call(Map parameters = [:], body) {
...
-                if (gitRepo) {
-                    checkoutFhcapCli(gitRepo, gitRef, true)
-                } else {
-                    //This is only here to get git clones to work correctly, and avoid 'Host key verification failed' errors
-                    checkoutFhcapCli()
-                    if (installLatest) {
-                        sh "gem install fhcap-cli --no-ri --no-rdoc"
-                    }
+        def configFile = parameters.get('configFile', "${env.WORKSPACE}/fhcap.json")
+        withEnv(["PATH+=/home/jenkins/bin", "RUBYOPT=-W0", "FHCAP_CFG_FILE=${configFile}"]) {
```
Comment (grdryn) [inline vars/fhcapNode.groovy:29]: does `"PATH+=/home/jenkins/bin"` work? I've never seen that syntax use for it before. I played around with it in a container, and I think it should instead be `"PATH+=:/home/jenkins/bin"`, does that sound right?

Comment (vchepeli) [reply vars/fhcapNode.groovy]: > does "PATH+=/home/jenkins/bin" work? I've never seen that syntax use for it before. I played around with it in a container, and I think it should instead be "PATH+=:/home/jenkins/bin", does that sound right?

Nope :)

Comment (ghost) [reply vars/fhcapNode.groovy:29]: > does "PATH+=/home/jenkins/bin" work? I've never seen that syntax use for it before. I played around with it in a container, and I think it should instead be "PATH+=:/home/jenkins/bin", does that sound right?

Nope :)
```
"PATH+=:/home/jenkins/bin"
:/home/jenkins/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/var/lib/jenkins/.local/bin
```

Comment (grdryn) [reply vars/fhcapNode.groovy:29]: Ah right, it's different behaviour in a pipeline than in a shell -- the shell also adds the new part as a suffix, where the pipeline adds it as a prefix.

**Review by grdryn — APPROVED** [review]

**Pull request merged**